command/repo.go

```diff
@@ -25,7 +25,7 @@ func init() {
 
 	repoCmd.AddCommand(repoCreateCmd)
 	repoCreateCmd.Flags().StringP("description", "d", "", "Description of repository")
-	repoCreateCmd.Flags().StringP("homepage", "h", "", "Repository home page URL")
+	repoCreateCmd.Flags().StringP("website", "w", "", "Repository home page URL")
```

Comment:
I don't believe that we should change existing `-h` flags to support `-h` across the board. Cobra already does a decent job of automatically treating `-h` as `--help` for commands that don't have their own `-h` flag. If a specific command wants to declare its own `-h` flag, I believe we should support it.

I have opened an alternate PR to address the `help` vs `--help` vs `-h` situation  #1221